scripts-dev/check_schema_delta.py

```diff
@@ -40,10 +40,39 @@ def main(force_colors: bool) -> None:
     exec(r, locals)
     current_schema_version = locals["SCHEMA_VERSION"]
 
-    click.secho(f"Current schema version: {current_schema_version}")
-
     diffs: List[git.Diff] = repo.remote().refs.develop.commit.diff(None)
 
+    for diff in diffs:
+        if (
+            diff.b_path == "synapse/storage/schema/__init__.py"
+            and diff.change_type == "M"
+        ):
+            # we've changed the schema file, let's check if the version has been changed
+            res = repo.git.show(
+                f"{repo.active_branch}:synapse/storage/schema/__init__.py"
+            )
```

Comment:
I think instead of looping through the diff we can just read the local file in the checkout? Which will be the merge commit?